codegen/src/main/java/software/amazon/awssdk/codegen/poet/auth/scheme/AuthSchemeSpecUtils.java

```diff
@@ -66,4 +73,30 @@ public boolean usesSigV4() {
         return AuthUtils.usesAwsAuth(intermediateModel);
     }
 
+    public String paramMethodName(String param) {
```

Comment:
nit: static

Reply:
Sure, is this a convention used by the java team in general?